pkg/config/nerdctl_config_applier_test.go

```diff
@@ -174,9 +176,76 @@ func Test_updateNerdctlConfig(t *testing.T) {
 			t.Parallel()
 
 			fs := afero.NewMemMapFs()
+			fc := &Finch{
```

Comment:
How about making fc one field in testCases struct so you can have all the test cases in one function instead of two?